tests/test_lib_user.py

```diff
@@ -199,7 +202,27 @@ def test_09_get_user_from_param(self):
                  "realm": self.realm2}
         user = get_user_from_param(param)
         self.assertEqual("{0!s}".format(user), "<cornelius.resolver1@realm2>")
-        
+
+        # test with splitAtSign set to '0'
```

Comment:
Why did you remove this one? Did it change or did you move it to the other tests?
I think then we can also delete it - not only comment it.

Reply:
This is a remnant from when i thought we could fix this all in one place...
But actually, the `splitAtSign` configuration isn't tested here (it is used in `get_user_from_param()`, not in `splig_user()`) so i would add some tests for the library functions as well.

Reply:
removed the commented code and added tests for `splitAtSign` in 3bcd19af